app/src/components/governance/ParticipantList.vue

```diff
@@ -141,6 +141,7 @@ h4 {
 .participant div {
   display: flex;
   align-items: center;
+  width: 7rem;
```

Comment:
this isn't really going to work on mobile

Reply:
some other idea how to accomplish the straight columns otherwise?

Reply:
i guess setting min and max widths with width being 100%

Reply:
lol https://github.com/luniehq/lunie/issues/4941